pkg/config/config.go

```diff
@@ -251,8 +255,11 @@ type Config struct {
 	// not enabled. Defaults to "http".
 	DefaultExternalScheme string
 
-	// DefaultExternal specifies whether internal traffic is encrypted or not.
+	// InternalEncryption specifies whether internal traffic is encrypted or not.
 	InternalEncryption bool
+
+	// InternalTrust specifies whether internal traffic is encrypted with mutual trust between peers.
```

Comment:
Does this mean that `internal-trust: true` and `internal-encryption: false` means that encryption is enabled?

Reply:
This is what I mean.

Reply:
I'm fine with that, but I might say "InternalTrust enforces InternalEncryption = true".

We might also do that in `NewConfigFromMap`.